.github/workflows/ci.yml

```diff
@@ -161,6 +161,7 @@ jobs:
           - "presto-oracle"
           - "presto-kudu"
           - "presto-phoenix,presto-iceberg,presto-druid"
+          - "presto-memsql -Dmemsql.license=${{ secrets.MemSqlLicense }}
```

Comment:
This needs to be conditional, otherwise the tests will fail for PRs or forks when the secret is not available.